### PR TITLE
Added fetch_quotes.py

### PR DESCRIPTION
**Pull request by Rahul664**
fetches quotes from zenquotes.io api

### Describe your change:

* [x] Added fetch_quotes.py under web_programming folder

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by Rahul664** [comment]
Hi,
I received this issue of pre-commit failed, can you help what is wrong and how can i resolve this issue.

File: web_programming/fetch_quotes.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+"""
+This file fetches quotes from the " ZenQuotes API ".
+It does not require any API key as it uses free tier.
+
+For more details and premium features visit:
+    https://zenquotes.io/
+"""
+
+import requests
+import pprint
```
Comment (rohanr18) [inline web_programming/fetch_quotes.py:10]:
```suggestion
import pprint

import requests
```

Comment (Rahul664) [reply web_programming/fetch_quotes.py]: Thank you for the edits, but what was the reason and how can I avoid this in future.

Comment (cclauss) [reply web_programming/fetch_quotes.py]: https://pycqa.github.io/isort

**Review by CaedenPH — CHANGES_REQUESTED** [review]
How is this an algorithm?

**Comment by Rahul664** [comment]
> 
It is not an algorithm but is an api functionality for web programming to fetch quotes. And if I am not wrong that is what the web programming folder files contain.

**Review by cclauss — APPROVED** [review]
Entries in the `web_programming` directory are a bit special because they are difficult to test and they more often look like how-to-do rather than being pure algorithms.  This one LGTM.

**Pull request merged**